go.mod

```diff
@@ -34,6 +34,7 @@ require (
 )
 
 require (
+	github.com/KimMachineGun/automemlimit v0.2.2
```

Comment:
Are we worried about this not being supported at some point since it seems very new still?

Reply:
These code are not being supported by anyone. They are opensource, so we just need to make sure we have viewed the source code and understand what it does. We run `go mod vendor` so the code is being preserved in our vendor directory in case it is removed from GitHub in the future.